oe_webtools_analytics/src/Event/WebtoolsImportSettingsEvent.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+declare(strict_types = 1);
+
+/**
+ * Webtools WebtoolsImportSettingsEvent Event.
```

Comment:
I would change the name: This doesn't import any settings.
Perhaps WebtoolsAnalytics ?